tests/jobs/localisation/example_case/scripts/common_functions.py

```diff
@@ -0,0 +1,548 @@
+"""
+Common functions used by the scripts: init_test_case.py and sim_field.py
+"""
+import copy
+import math
+
+import gstools as gs
+import numpy as np
+import yaml
+
+import xtgeo  # isort: skip
+
+# pylint: disable=missing-function-docstring, too-many-locals, invalid-name
+# pylint: disable=bare-except, raise-missing-from
```

Comment:
This is not needed if `get_seed` is fixed as suggested below.

Reply:
OK. Fixed